keps/sig-node/2862-fine-grained-kubelet-authz/README.md

```diff
@@ -799,11 +806,28 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+We have tested the following upgrade scenarios manually:
+
+|Scenario| Result |
+| -------|--------|
+| Upgrade both kubelet and kube-apiserver so that feature gate is enabled in both. | workloads and kube-apiserver are able to reach kubelet|
+| Upgrade only kubelet to enable the feature-gate | workloads and kube-apiserver are able to reach kubelet |
+| Updrade only kube-apiserver to enable the feature-gate | workloads and kube-apiserver are able to reach kubelet |
```

Comment:
For skew policy, n version apiserver may work with n-3 version kubelet. https://kubernetes.io/releases/version-skew-policy/#kubelet

IIUC, this can meet the policy.

Reply:
Yes it will because we have a fallback to the old behavior.